demisto_sdk/commands/common/errors.py

```diff
@@ -1728,8 +1729,15 @@ def pack_metadata_non_approved_usecases(non_approved_usecases: set) -> str:
     @staticmethod
     @error_code_decorator
     def pack_metadata_non_approved_tags(non_approved_tags: set) -> str:
-        return f'The pack metadata contains non approved tags: {", ".join(non_approved_tags)}' \
-               f'The list of approved tags can be found in https://xsoar.pan.dev/docs/documentation/pack-docs#pack-keywords-tags-use-cases--categories'
+        return f'The pack metadata contains non approved tags: {", ".join(non_approved_tags)}. ' \
+               'The list of approved tags for each marketplace can be found on' \
+               'https://xsoar.pan.dev/docs/documentation/pack-docs#pack-keywords-tags-use-cases--categories'
```

Comment:
Maybe it would be better to define this address at the top of the file, so it'll be easier to find when/if it needs updating?